test/bigtest/network/config.js

```diff
@@ -482,7 +482,6 @@ export default function configure() {
           id: '44a21d17-a666-4f34-b24d-644f8ed1537b',
           hrid: 'ho00000111',
           formerIds: [],
-          instanceId: '7a8f9775-8de3-4308-8323-b055c104e0f3',
```

Comment:
Why remove this? `instanceId` is a legit part of a `holdings-storage/holdings` response.

Reply:
Taras and I have checked to confirm that particular `instanceId` isn't used anywhere in any of the test suites so it's probably just a placeholder. Maybe @mkuklis can help us understand why it's there.

Reply:
@minkimcello since this is a valid attribute as @zburke mentioned please keep it around.

Reply:
I asked Min to remove remove it because it's not actually used for anything. That attribute is generated by mirage factories so it's literally code that has no impact on the tests or the app. But we can put it back in if you prefer.

Reply:
[Done](https://github.com/folio-org/ui-inventory/pull/1110/commits/e829c4aefb0c3e125c5a734b50aad0cc4d942f02).

Reply:
I'll take a look at this with min. There might be a problem with how the payload is serialized/deserialized

Reply:
We should still remove this because it's dead code but solution to this is now in #1120